### PR TITLE
Fix cache control

### PR DESCRIPTION
**Pull request by vapaj**
Don't send empty cache-control header

### CODE REVIEW

File: packages/components/src/Persona.purs [hunk 1]
```diff
@@ -47,7 +47,7 @@ getUser invalidateCache uuid token =
   where
     headers =
       { authorization
-      , cacheControl: fromMaybe mempty maybeCacheControl
+      , cacheControl: toNullable maybeCacheControl
```
Comment (ae-mo) [inline packages/components/src/Persona.purs]: So this way the `cache-control` header doesn't appear?

Btw wasn't this introduced in a recent PR I reviewed?

Comment (ae-mo) [reply packages/components/src/Persona.purs]: Oh no that was in `vetrina`

Comment (vapaj) [reply packages/components/src/Persona.purs]: It was yeah, it's a shared module though 😄

Comment (vapaj) [reply packages/components/src/Persona.purs]: Yes, if we set the `cacheControl` to `null`, it won't appear in the request

**Pull request merged**